system/mp/mpbuff.hpp

```diff
@@ -78,7 +78,7 @@ class mp_decl CMessageBuffer: public MemoryBuffer
                 data.append(", ");
             data.append((unsigned)(byte)toByteArray()[i]);
         }
-        return buf.appendf("CMessageBuffer(%8X) tag=%d, sender=%s, replytag=%d, size== %d, data head = %s", (int)(long)toByteArray(), (int)tag, sender.getUrlStr(ep).str(), (int)replytag, length(), data.str());
+        return buf.appendf("CMessageBuffer(%P) tag=%d, sender=%s, replytag=%d, size== %d, data head = %s", toByteArray(), (int)tag, sender.getUrlStr(ep).str(), (int)replytag, length(), data.str());
```

Comment:
Is this supposed to be lower-case 'p' for 'pointer' type? I see there is also a warning thrown by smoketest on this line.

Reply:
yes, fixed.